test_regress/t/t_var_sc_bv.cpp

```diff
@@ -0,0 +1,224 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//
+// This file ONLY is placed into the Public Domain, for any use,
+// without warranty, 2008 by Lane Brooks.
```

Comment:
```suggestion
// without warranty, 2022 by Wilson Snyder.
```

or your name, either's ok.